molecule/static_cluster/prepare.yml

```diff
@@ -7,4 +7,17 @@
         name:
           - sudo
           - iproute
-        state: present
\ No newline at end of file
+        state: present
+    - name: "Create {{ activemq.service_name }} service user"
+      become: yes
+      ansible.builtin.user:
+        name: amq-broker
+        home: /opt/amq/
+        system: yes
+        create_home: no
+    - name: Set ownership on nfs mounted parent
+      file:
+        path: /opt/amq/amq-broker/
```

Comment:
@guidograzioli is there not a amq_home variable to be used here? This hard coded path will be complicated the minute user use a different folder for AMQ.

Reply:
it is in a test preparation, before the role executes (with the defaults)